tests/common/requirements.txt

```diff
@@ -1,4 +1,6 @@
 -c ../../constraints.txt
+torch
```

Comment:
As it used torch and transformers, i suggest to add this test to tests/torch

Reply:
Done. From now, we'll test the common code in `tests/torch`.